resources/windows/msi/service.wxs.erb

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+<!-- include the set of service files -->
+
+  <%- service_files.each do |service| -%>
+    <?include $(sys.SOURCEFILEDIR)include/<%=  service.path %>' ?>
```

Comment:
Ok - perhaps the file is superfluous so perhaps we should drop it altogether and just include the wxs files.